BookAPI/Program.cs

```diff
@@ -1,17 +1,85 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace BookAPI
 {
     public class Program
     {
+        public static List<BookModel> bookList = new List<BookModel>();
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Book Exchange");
+            BookModel newBook = new BookModel();
+            string bookName, bookOwner;
+            bool bookAvailability = false;
+            do
+            {
```

Comment:
Dont have to loop it, if fails just show exception and end program?
Similar to how api is expected to work